README.rst

```diff
@@ -59,18 +57,21 @@ specify all steps required to produce a model: input dependencies including data
 and output information to be saved. See the quick start section below or
 the `Get Started <https://dvc.org/doc/get-started>`_ tutorial to learn more.
 
-Quick start
-===========
+Quick start (CLI)
+=================
+
+    Please read our `Command Reference https://dvc.org/doc/command-reference>`_
+    for a full reference.
 
-Please read `Get Started <https://dvc.org/doc/get-started>`_ guide for a full version. Common workflow commands include:
```

Comment:
Why get rid of this? I know it's above, but considering we will want to swap places, it would be good to link to get started early.

Reply:
WIP ⏳

Reply:
This was removed because there's 2 Quick starts now, this one is specific to command line so I linked to the Command Reference docs instead.